superset-frontend/src/views/CRUD/chart/ChartList.tsx

```diff
@@ -364,7 +364,7 @@ function ChartList(props: ChartListProps) {
       Header: t('Search'),
       id: 'slice_name',
       input: 'search',
-      operator: 'name_or_description',
+      operator: 'all_text_chart',
```

Comment:
Looks like arg_names share a global namespace? If so, we should add the the resource name to other filters so there's no confusion/collision.